### DISCUSSION TITLE
v20.0.13

### DISCUSSION THREAD

**Discussion by Flyingmana** [OpenMage/magento-lts#1799] [Releases]
## v20.0.13 - 2021-08-26

2 security updates

CVE-2021-32758 - GHSA-26rr-v2j2-25fh - Layout XML Arbitrary Code Fix
CVE-2021-32759 - GHSA-xm9f-vxmx-4m58 - Data Flow Sanitation Issue Fix

more updates:

#1735 Fixed origData is empty in a quote
#1667 Removed orphan directory and code of compiler and downloder.
#1355 Removed unused variable
#1525 Fixed incorrect datetime in block.

and merged changes from v19.4.15

<hr /><em>This discussion was created from the release <a href='https://github.com/OpenMage/magento-lts/releases/tag/v20.0.13'>v20.0.13</a>.</em>

**Comment by addison74** [comment]
Good job. Let's thank all those who made this possible. After 1 year Magento 1 platform is still alive thanks to this wonderful project.
